libcxx/include/string

```diff
@@ -3446,6 +3447,14 @@ inline _LIBCPP_CONSTEXPR_SINCE_CXX20 void basic_string<_CharT, _Traits, _Allocat
     _NOEXCEPT_(!__alloc_traits::propagate_on_container_swap::value || __is_nothrow_swappable_v<allocator_type>)
 #endif
 {
+
+#if _LIBCPP_STD_VER >= 26
```

Comment:
You can't do this unless the allocator is replaceable, too.  (Usually it will be, but not always)

Reply:
That goes to show how little i know about allocators!